coalib/processes/Processing.py

```diff
@@ -733,7 +757,12 @@ def execute_section(section,
                              results (bear names are key) as well as the
                              file dictionary.
     """
-    if debug:
+    debug_bears = (False
+                   if 'debug_bears' not in section or (
+                       section['debug_bears'].value == 'False') else
+                   typed_list(str)(section['debug_bears']))
```

Comment:
does it have to be a typed list?

Reply:
I use the typed_list so that it will split the comma separated value.

Reply:
Any other suggestion will be helpfull for me :grinning:

Reply:
Ahhh nvm... without `typed_list` you don't get a solid list type you can use in Python like a normal list... Nvm, you're doing it totally correct :+1: